sound/soc/sof/control.c

```diff
@@ -114,6 +114,30 @@ int snd_sof_volume_put(struct snd_kcontrol *kcontrol,
 	return change;
 }
 
+int snd_sof_volume_info(struct snd_kcontrol *kcontrol,
+			struct snd_ctl_elem_info *uinfo)
```

Comment:
one line

Reply:
hmm, this in my knowledge doesn't fit one line? Or did we change the rules, my emacs in linux mode doesn't like this in 1 line...

Reply:
The new limit is 100chars. Is it more than that?

Reply:
ok yes... took me a while to find the config in emacs :) yes should fit to 100, so will change. thanks!